src/main/resources/static/_shared.yml

```diff
@@ -3480,6 +3490,7 @@ components:
               format: date-time
             assessmentId:
               type: string
+              description: If type is 'Additional', provides the PlacementApplication ID. If type is 'Initial' this field shouldn't be used.
```

Comment:
is it worth noting here that this is a temporary complexity, as the separate PlacementApplication entity is being removed and we'll only have PlacementRequests "soon"?

Reply:
I suspect this may be permanent as we'd still (presumably) want to differentiate between these two types even in the new data model (although we will probably do this using subtypes I guess!)